packages/sources/tradermade-test/src/endpoint/forex-router.ts

```diff
@@ -0,0 +1,21 @@
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/meta'
+import { PriceEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { BatchEndpointTypes } from '../price-utils'
+import { httpTransport, inputParameters } from './forex'
+import { wsTransport } from './forex-ws'
+
+const transports = {
+  websocket: wsTransport,
+  batch: httpTransport,
+}
```

Comment:
Could you update the names to `ws` and `rest`, respectively? I believe that's what EngOps will be configuring the feeds to.